output/slide-deck.qmd

```diff
@@ -530,62 +428,85 @@ git config --list --global
+e.g.:
+
+git config --global user.email "DaniMori\@users.noreply.github.com"
 :::
 
-## Let's make some changes
+## Let's make some changes {.smaller}
+
+<br>
 
--   Delete unnecessary folders / files
+-   Delete unnecessary folders
 
 ::: {.nonincremental .fragment}
--   Edit `README.md`:
+-   Edit "README.Rmd":
 
-    -   Change \<repository-name\> by your repo (lines 2 & 113)
+    -   Change `<repository-name>` by your repo (lines 12-14)
 
-    -   Change \<repository-url\> by yours (line 66)
+    -   Change `<username>` by your GitHub user name (lines 16-18)
 
-    -   Edit "License" section (lines 8-32)
+    -   Edit description in section "Repository" (lines 23-24)
 
-    -   Edit "description" (line 4)
+    -   Delete description of unused folders (lines 158-181)
 :::
 
+-   Knit "README.md"
+
 ::: notes
-IMPORTANT!! Do not delete `.Rprofile`, `renv.lock` or the `renv` folder
+Delete folders "apps", "doc", "notebooks", "output", and/or "R" (give
+flexibility, depending on what they plan to add to their project)
 
-Delete: `www`
+(Note that this results in deleting the file(s) within, not the folder; in this
+case, the ".kitkeep" files. git has no notion of folders, only files.)
 
-Note that this results in deleting the file(s) within, not the folder
+Do NOT delete: "src", "dat" (both necessary later on, if they use the script and
+dataset provided), or "renv".
 
-Other candidates to delete? (Up to each one)
+IMPORTANT!! NEVER delete ".Rprofile", "renv.lock" or the "renv" folder
 
 (NEXT)
 
-README file: Information for "visitors" of your repository. Useful if you plan
-to make it public at some point. **Give some time to get familiar with it!**
+We will create a "README" file for our new project. Information for "visitors"
+of your repository. Useful if you plan to make it public at some point. **Give
+some time to get familiar with it!**
+
+We will edit "README.Rmd" (NOT "README.md"), then knit it to generate our own
+"README.md" file that will overwrite the one with the template information.
+
+NOTE: There are other sections that can be personalized (see TODOs throughout).
 
-Repo URL must contain the `.git` suffix!!
+NOTE: The lines where the "Description of unused folders" is may change when
+editing the "repository description" in the previous step.
 
-License:
+IMPORTANT: HIGHLIGHT this is *just a text file*; git versions (easily) text
+files, and keeps track of the changes. Scripts (and ".R" files in general),
+notebooks, and Quarto documents are **also text files** (just characters).
 
--   Change "template" by "project" (line 8)
+(NEXT)
 
--   Delete bullet points
+Press "knit" button to generate "README.md". This overwrites the template's own
+"README.md" file, creating one for their own repository.
 
--   Convert "citation" to "normal text"
+It has installation instructions and all the necessary information (e.g.,
+license file; a brief comment on licensing may be in place here).
 :::
 
 # Versioning in git
```

Comment:
I find the image here a bit confusing even for me unfortunately... I fear that we might spend too long explaining it. What about a simpler one, like:
https://geo-python.github.io/site/2018/_images/Git_illustration.png
from https://geo-python.github.io/site/2018/lessons/L2/git-basics.html
No worries if you want to keep it! Just have to try to keep the discussion a bit shorter!

Reply:
Not very intuitive, right. But I'd rather use one that makes (for now) no reference to pushing to/pulling from remote repositories, to keep it down to "versioning" at this point (later on we introduce "synchronizing").

We can use your proposal for the "synchronizing" section. Check out these alternatives:

https://www.freecodecamp.org/news/content/images/2020/08/git-basic-workflow-codesweetly.png
(from https://www.freecodecamp.org/news/git-and-github-overview/)

https://3.bp.blogspot.com/-JOstLszmx1Y/VqnTd7vfUjI/AAAAAAAArJ0/r24DWMC_pUo/s1600/git-staging-diagram.png
(from https://hanxue-it.blogspot.com/2016/01/why-git-commit-has-two-stages.html)

In any case, you can consider it an "introduction" slide and skip it (leaving it for later use as reference for attendees). It might be easier though to just use something purely illustrative about versioning, even if it's not git-related (and forget about the "use as reference").